vault/logical_system_paths.go

```diff
@@ -1196,7 +1335,18 @@ func (b *SystemBackend) leasePaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ListOperation: &framework.PathOperation{
 					Callback: b.handleLeaseLookupList,
-					Summary:  "Returns a list of lease ids.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"keys": {
+									Type:        framework.TypeCommaStringSlice,
+									Description: "A list of lease ids",
+									Required:    true,
```

Comment:
`logical.ListResponse` only has `keys` when there's more than one item
https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/sdk/logical/response.go#L140-L142
```suggestion
									Required:    false,
```